### DISCUSSION TITLE
Status Indicator pattern

### DISCUSSION THREAD

**Discussion by wilfredomerced** [twilio-labs/paste#1664] [General] [answered]
Is there a way to see the status of a pattern/component thats being currently worked on? or be able to see rough timeline of when it might be available? In the paste table documentation its mentioned that it's being worked on (https://paste.twilio.design/components/table/#table-with-status-indicators)

**Comment by gloriliale** [comment, marked as answer]
Hi @wilfredomerced! Our website has a [Roadmap section](https://paste.twilio.design/roadmap/) that shows what we're working on for the next few releases- open the disclosures to see more details.. For status indicator specifically, you'll see it under the 2021.08.24 release!
